packages/react-utilities/package.json

```diff
@@ -27,13 +27,14 @@
   },
   "devDependencies": {
     "@fluentui/eslint-plugin": "^1.0.0-beta.2",
+    "@fluentui/jest-serializer-merge-styles": "^8.0.0-beta.6",
```

Comment:
In theory react-utilities shouldn't need this since it's only used for converged, and merge-styles is v8 only.